File: SingleAPIExample/python/Approve.py
import splitit

username, password, api_key = (
    "username",
    "password",
    "api_key",
)

api_client = splitit.ApiClient(splitit.Configuration(username, password, api_key, sandbox=True))


def approve_installment_plan(ipn):
    resp = api_client.InstallmentPlanApi.installment_plan_approve(
        splitit.ApproveInstallmentPlanRequest(
            installment_plan_number=ipn,
        )
    )
    return resp

File: SingleAPIExample/java/Approve.java
import com.splitit.*;
import com.splitit.sdk.model.*;
import java.math.BigDecimal;


public class Approve {
    private String apiKey, username, password;
    public Approve(String username, String password, String apiKey){
        this.username = username;
        this.password = password;
        this.apiKey = apiKey;
    }

    public void example(String ipn) throws ApiException{
        ApiClient apiClient = new ApiClient()
            .sandbox(true)
            .username(username)
            .password(password)
            .apiKey(apiKey);

        var result = apiClient.getInstallmentPlanApi().installmentPlanApprove(
            new ApproveInstallmentPlanRequest()
            .installmentPlanNumber(ipn)
        );
        System.out.println(result);
    }
}
